auth/routes.py

```diff
@@ -0,0 +1,18 @@
+from flask import Blueprint, request, jsonify
+from auth.utils import authenticate, generate_token
+
+auth_bp = Blueprint('auth', __name__)
+
+@auth_bp.route('/login', methods=['POST'])
+def login():
+    data = request.get_json()
+    user = authenticate(data.get('email'), data.get('password'))
```

Comment:
data might be null here
```suggestion
def login():
    data = request.get_json() or {}
    user = authenticate(data.get('email'), data.get('password'))
```